test/units/galaxy/test_collection.py

```diff
@@ -198,6 +199,48 @@ def manifest(manifest_info):
             yield fake_file, sha256(b_data).hexdigest()
 
 
+@pytest.fixture()
+def server_config(monkeypatch):
+    monkeypatch.setattr(C, 'GALAXY_SERVER_LIST', ['server1', 'server2', 'server3'])
+
+    default_options = dict((k, None) for k, v in SERVER_DEF)
+
+    server1 = dict(default_options)
```

Comment:
Did you mean
```suggestion
    server1 = default_options.copy()
```

Reply:
They are both ways of making shallow copies.